### PR TITLE
[DDO-2491] `thelma argocd sync --refresh-only`

### PR DESCRIPTION
**Pull request by jack-r-warren**
There's a test too

### CODE REVIEW

File: internal/thelma/ops/sync/sync.go [hunk 1]
```diff
@@ -33,13 +33,19 @@ type syncer struct {
...
+func (s *syncer) Sync(releases []terra.Release, maxParallel int, refreshOnly bool, options ...argocd.SyncOption) (map[terra.Release]*status.Status, error) {
 	var jobs []pool.Job
 
 	waitHealthyTimeout := s.extractWaitHealthy(options)
 
-	optionsNoWaitHealthy := withOption(options, func(options *argocd.SyncOptions) {
+	commonOptions := withOption(options, func(options *argocd.SyncOptions) {
 		options.WaitHealthy = false
+	}, func(options *argocd.SyncOptions) {
+		if refreshOnly {
```
Comment (choover-broad) [inline internal/thelma/ops/sync/sync.go:44]: set this in sync_command.go so we don't have to add a new parameter to this function?

**Review by choover-broad — APPROVED** [review]
🚤 🎉

File: internal/thelma/ops/sync/sync.go (hunk 1 above)
Comment (jack-r-warren) [reply internal/thelma/ops/sync/sync.go:44]: That's what I did first, but then I can't set the text output at the bottom of this function, and it always says "1/1 services synced"

Comment (choover-broad) [reply internal/thelma/ops/sync/sync.go:44]: IMO that's a worthwhile tradeoff but up to you!

Comment (jack-r-warren) [reply internal/thelma/ops/sync/sync.go]: fair enough

**Pull request merged**